README.md

```diff
@@ -193,8 +193,11 @@ cluster as service `kafka-connect` on port `8083`.
 
 ### Using Kafka Connect with additional plugins
 
-Our Kafka Connect images contain by default only the `FileStreamSinkConnector` and `FileStreamSourceConnector` connectors.
-If you want to use other connectors, you can:
+Our Kafka Connect Docker images contain by default only the `FileStreamSinkConnector` and 
+`FileStreamSourceConnector` connectors which are part of the Apache Kafka project.
+
+Kafka Connect will automatically load all plugins / connectors which are present in the `/opt/kafka/plugins` 
```

Comment:
I'm not using spaces before and after the "/" character. But it's just a choice. Let's do it in the same way, with or without spaces.

Reply:
Fixed.